tests/forms_tests/tests/test_formsets.py

```diff
@@ -617,6 +620,32 @@ def test_formsets_with_ordering(self):
             {'votes': 900, 'ORDER': 2, 'choice': 'Fergie'},
         ])
 
+    def test_formsets_with_order_custom_widget(self):
+        class FormSet(BaseFormSet):
+            ordering_widget = HiddenInput
+
+        class ArticleForm(ModelForm):
+            class Meta:
+                model = Article
+                fields = ('content', )
+
+        ChoiceFormSet = formset_factory(ArticleForm, formset=FormSet, can_order=True)
+        initial = [{'choice': 'Calexico', 'votes': 100}, {'choice': 'Fergie', 'votes': 900}]
+        formset = ChoiceFormSet(initial=initial, auto_id=False, prefix='choices')
```

Comment:
I don't think that creating a custom `ArticleForm` or setting `initial` data is required for this test, we should be able to simplify this, e.g.
```python
def test_formsets_with_order_custom_widget(self):
    class FormSet(BaseFormSet):
        ordering_widget = HiddenInput

    ArticleFormSet = formset_factory(ArticleForm, formset=FormSet, can_order=True)
    formset = ArticleFormSet(auto_id=False, prefix='articles')
    self.assertHTMLEqual(
        ...
```